docs/reference/examples/mission_impact.json

```diff
@@ -0,0 +1,29 @@
+{
+  "namespace": "ssvc",
+  "version": "2.0.0",
+  "name": "Mission Impact",
+  "description": "Impact on Mission Essential Functions of the Organization",
```

Comment:
Similar question to my comment on *Automatable*, can we embed a link to the presidential policy directive here? https://www.cisa.gov/sites/default/files/2023-01/ppd-21-critical-infrastructure-and-resilience-508_0.pdf

Reply:
I don't want to have to bump a version in an SSVC object when a URL changes on a site we don't control. The description that goes into the json should be fairly minimalist. The documentation surrounding it can have whatever details we want to add.